### PR TITLE
Separate reference docs into different block

### PR DESCRIPTION
**Pull request by LittleLittleCloud**
## Purpose

* ...
#175 
## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Review by luisquintanilla — COMMENTED** [review]
@LittleLittleCloud would these changes be optional?

**Review by luisquintanilla — COMMENTED** [review]
@LittleLittleCloud can we remove the README changes. I've already added them to #188. Thanks

File: infra/main.bicep [hunk 1]
```diff
@@ -77,6 +67,9 @@ param keyVaultResourceGroupName string = ''
 @description('Name of the Azure Log Analytics workspace')
 param logAnalyticsName string = ''
 
+@description('Location of the resource group for the OpenAI resources')
```
Comment (luisquintanilla) [inline infra/main.bicep:70]: @LittleLittleCloud we're defaulting to using the resource group location here? Will this still allow users to choose a different location for their resources?

Comment (LittleLittleCloud) [reply infra/main.bicep:70]: oops sorry

**Review by luisquintanilla — APPROVED** [review]

**Pull request merged**